app/src/main/java/com/scv/slackgo/activities/LocationActivity.java

```diff
@@ -269,9 +275,11 @@ public void onClick(View v) {
         saveLocationButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                channels = ChannelListHelper.channelsFromTextViewString(channelsTextView.getText().toString());
+                channelsByName = ChannelListHelper.channelsFromTextViewString(channelsTextView.getText().toString());
```

Comment:
I think is better if we only save the list of channels. Then, when we need to create the list of channels' name, we are able to use CollectionUtils.collect.